__tests__/app.test.js

```diff
@@ -191,3 +191,40 @@ describe("GET api/articles,", () => {
       });
   });
 });
+describe("GET api/articles/:article_id/comments", () => {
+  test("status(200) returns an array of comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
If using a forEach in the test suite like this, we need to check the array length to avoid a false positive